lib/components/artist/articles/index.js

```diff
@@ -42,7 +42,7 @@ export default Relay.createContainer(Articles, {
   fragments: {
     articles: () => Relay.QL`
       fragment on Article @relay(plural: true) {
-          id
+          __id
```

Comment:
From now on we should always use `__id` for these purposes, because:

* Relay will implicitly query for this field, if it exists, anyways, so might as well save some bandwidth.
* This ID is guaranteed to be unique, whereas `id` might exist multiple times across types.